server/tests/participant.form.test.js

```diff
@@ -264,21 +265,33 @@ describe('Participants Service', () => {
       regions,
     });
 
-    const filteredParticipants = allParticipants
-      .filter((item) => (
-        evaluateBooleanAnswer(item.interested)
-        && evaluateBooleanAnswer(item.crcClear)));
-
-    expect(res.data.map((item) => (Object.keys(item)))).toEqual(
-      filteredParticipants.map(() => ([
-        'id',
+    const mapRawToEmployerColumns = (a) => {
+      const employerColumns = [ // Fields expected to be returned to employers
         'firstName',
         'lastName',
         'postalCodeFsa',
         'preferredLocation',
         'nonHCAP',
-      ])),
-    );
+        'statusInfo',
+        'progressStats',
+      ];
+      return a.map((i) => (
+        Object.keys(i)
+          .filter((k) => employerColumns.includes(k))
+          .reduce((o, k) => ({ ...o, [k]: i[k] }), { nonHCAP: undefined })
+      ));
+    };
+    const trimIds = (a) => a.map((i) => (
+      Object.keys(i)
+        .filter((k) => !['id', 'callbackStatus'].includes(k)) // TODO: Should not ignore callback status
```

Comment:
Is this TODO still relevant?

Reply:
Still a TODO. Not sure how we should solve this besides replicating the exact algo used on server. Currently this is:
```
const callbackStatus = (participant) => {
  const d = new Date(participant.updated_at);
  if (d instanceof Date && !Number.isNaN(d.getTime())) {
    if (d < new Date('2021-01-01T00:00:00-08:00')) {
      return 'Primed';
    }
  }
  return 'Available';
};
```
Think we're good to keep ignoring this for the time being. The properties that are being compared in this test are way stricter than those in other tests which are just looking at keys. This is what I was looking to address with the `expect` extension, but did not have the time to finish.

Reply:
Could you log a ticket in the backlog for this? A spike or a task, doesn't really matter - just don't want to lose line of sight on this